apps/backend/app/routers/world_entities.py

```diff
@@ -0,0 +1,59 @@
+from fastapi import APIRouter, HTTPException, Depends
+from sqlalchemy.orm import Session
+from typing import List
+from app.schemas import character as character_schema
+from app.schemas.world_entity import WorldEntity
+from app.crud.world_entities import get_entities_by_story_id, get_entity_by_id
+from app.crud.scenes import get_scene_by_uuid
+from app.services.world_entity_service import WorldEntityService
+from app.db.session import get_db
+
+
+router = APIRouter(
+    prefix="/world_entities",
+    tags=["world_entities"],
+)
+
+@router.get("/{story_id}", response_model=List[WorldEntity])
```

Comment:
to distinguish it from the endpoint below
```suggestion
@router.get("/stories/{story_id}", response_model=List[WorldEntity])
```